### ISSUE TITLE
request: add pauses

### ISSUE THREAD

**Issue by glsf91** [pndurette/gTTS#102]
When using 2 small sentences the pause is not long enough in google translate and also with this (great) tool. 
Can you not make it possible to add a configurable pause.  The program have to add the pause by itself because google is not doing this.
For example:
This is sentence 1 <@pause:5> This is sentence 2 after 5 seconds pause.

So the program has to: 
- translate first part at google
- generate a 5 sec pause 
- translate second part at google
- combine this in the output file

Define pause in ms would be better, but depends on the solutions.

That will be a great feature I think.

**Comment by pndurette** [comment]
I agree! I've been asked that a lot, and I've decided to give it a shop, soon-ish (working on `2.0.0` for the moment). It would probably work using Speech Synthesis Markup Language ([SSML](https://www.w3.org/TR/speech-synthesis11/)) or an SSML-like syntax. So no need for something like `<@pause:5>`. In SSML that would be `<break time="5s"/>` 😃

**Comment by glsf91** [comment]
That is great. 
It was just an example. :-)

**Comment by ekcloud** [comment]
Any updates on this issue?
SSML integration will make this tool much more powerful and useful!

**Comment by Maneshgpai** [comment]
Voting for this, SSML as input will be awesome. Thanks for the great work pndurette.

**Comment by daryl-sen** [comment]
Dying for an SSML update! I think it would make many people very happy!

**Comment by nikhileshkoshti** [comment]
Any update. Whether SSML supported or not?

**Comment by shashank333** [comment]
you could even import time for that
`time.sleep(1)`
in between the sentences to sleep

**Comment by SiliconXu** [comment]
@pndurette Hi It's been a few years. I wonder if this feature has been implemented. (I could not find it on the document)
Thanks!
